topics/sample-topic.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: "Sample Title"
+description: "Sample description of the topic covered in the document."
+tags: ["Tag1", "Tag2", "Tag3"]
+excerpt: "This is a brief summary of the topic, designed to include relevant keywords for SEO."
+featuredImage: 'https://example.com/featured-image.png'
```

Comment:
Is there a possibility that featured image would be more than one? If so, we might want to make this an array of images

Reply:
I think we'll use just one image per topic as thumbnail